libs/action.js

```diff
@@ -33,6 +30,11 @@ let action = {
         `Element '${selector} does not contain text.`);
   },
 
+  waitAndRefreshPage(waitTime = 0) {
+    browser.refresh();
+    browser.pause(waitTime);
```

Comment:
Maybe you should wait some element not just sleep?

Reply:
Remove this method. I've never used it